CHANGELOG.md

```diff
@@ -4,7 +4,17 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.11.0 (2023-05-12)
+
+This release contains new features and bug fixes since the 2.10.3 release.
+We deem it moderate priority for upgrading.
+
+This release includes these noteworthy features:
+* Compressed hypertable enhancements:
+  * UPDATE/DELETE support
+  * ON CONFLICT DO UPDATE
+* Join support for hierarchical Continuous Aggregates
+* Performance improvements
```

Comment:
Removing this line cause `performance improvements` is too generic - if we have a notable performance improvement to mention, we should do so explicitly (e.g. `compression is 3123 times faster` or `queries over compressed chunks that filtrer by the segment by column are 2897 times faster`)

```suggestion
```